src/main/webapp/view/discussion/discussion.js

```diff
@@ -91,11 +95,14 @@ async function postAndReload(
 async function loadDiscussion() {
   // Clear any existing comments before loading.
   ELEMENT_DISCUSSION.textContent = '';
+  currentRootDiscussionComments = [];
 
   const comments = await fetchDiscussion();
   const preparedComments = prepareComments(comments);
   for (const comment of preparedComments) {
-    ELEMENT_DISCUSSION.appendChild(new DiscussionComment(comment));
+    const element = new DiscussionComment(comment);
```

Comment:
This name could be more descriptive element is general

Reply:
changed to `commentElement`